src/content/blog/astro-db-deep-dive.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "Astro DB: A Deep Dive"
+description: "Yesterday we launched a fully-managed SQL database service designed exclusively for the Astro web framework. Let's dive into the implementation details of Astro DB: how it works, why we built it, and why we're adopting libSQL."
```

Comment:
Here we should use *fully managed*, fully ends in `-ly` and that means it doesn't get hyphenated. I am aware it's also being used with a hyphen in the landing page, so I'll take a look and see if we still have time to change that there as well.
```suggestion
description: "Yesterday we launched a fully managed SQL database service designed exclusively for the Astro web framework. Let's dive into the implementation details of Astro DB: how it works, why we built it, and why we're adopting libSQL."
```